### PR TITLE
Add "build" input option for workflow_dispatch

### PR DESCRIPTION
**Pull request by MonicaisHer**
Trigger manual test with setting the input to `True to enable local build` on my local repo's action: https://github.com/MonicaisHer/edgex-snap-testing/actions/runs/2801769488

### CODE REVIEW

**Comment by MonicaisHer** [comment]
Trigger manual test with setting the input to `True to enable local build` on my local repo's action: https://github.com/MonicaisHer/edgex-snap-testing/actions/runs/2797660133

File: .github/workflows/snap-testing.yml [hunk 1]
```diff
@@ -56,10 +77,21 @@ jobs:
...
 
+      - name: Build snap
+        if: ${{ github.event_name == 'workflow_dispatch' && inputs.localBuild }}
+        uses: ./build
+        id: build
+        with:
+          repo: ${{matrix.repo}}
+
+      - name: Checkout the local actions
+        uses: actions/checkout@v2
```
Comment (farshidtz) [inline .github/workflows/snap-testing.yml:88]: There is a step named "Checkout the local actions again". I believe that is from before when there was a build stage.
You can remove this and move "Build snap" to before that existing, second checkout.

File: .github/workflows/snap-testing.yml [hunk 2]
```diff
@@ -56,10 +77,21 @@ jobs:
       - name: Checkout the local actions again
         uses: actions/checkout@v2
 
+      - name: Build snap
+        if: ${{ github.event_name == 'workflow_dispatch' && inputs.localBuild }}
```
Comment (farshidtz) [inline .github/workflows/snap-testing.yml:81]: Is checking the event name necessary?

**Review by farshidtz — CHANGES_REQUESTED** [review]
Looks good. Please see my minor comments.

File: .github/workflows/snap-testing.yml (hunk 2 above)
Comment (MonicaisHer) [reply .github/workflows/snap-testing.yml]: You are right, this is unnecessary. Updated.

**Review by farshidtz — APPROVED** [review]

**Pull request merged**